std_msgs/Header header # Message header including timestamp of detection & img frame
Scene[] detected_scene       # Array of detected objects
